Update deploy.md (#95838)
Azure/azure-content: articles/azure-functions/start-stop-vms/deploy.md

    :::image type="content" source="media/deploy/schedule-recurrence-property.png" alt-text="Configure the recurrence frequency for logic app":::

> 1. In the designer pane, select **Function-Try** to configure the target settings and click the "</> Code view" button at the top menu to edit the code for the "Function-Try" element. In the request body, if you want to manage VMs across all resource groups in the subscription, modify the request body as shown in the following example.

Reviewer: @TomaszWieczorkowski thanks for your help updating this. UI elements are in bold:
```suggestion
1. In the designer pane, select **Function-Try** to configure the target settings and select the **</> Code view** button at the top menu to edit the code for the **Function-Try** element. In the request body, if you want to manage VMs across all resource groups in the subscription, modify the request body as shown in the following example.
```

> 1. In the designer pane, select **Function-Try** to configure the target settings and click the "</> Code view" button at the top menu to edit the code for the "Function-Try" element. In the request body, if you want to manage VMs across all resource groups in the subscription, modify the request body as shown in the following example.

Reviewer 2: ```suggestion
1. In the designer pane, select **Function-Try** to configure the target settings and then select the **</> Code view** button in the top menu to edit the code for the **Function-Try** element. In the request body, if you want to manage VMs across all resource groups in the subscription, modify the request body as shown in the following example.
```
Suggested changes for Microsoft style
